static/js/dialog_widget.js

```diff
@@ -91,6 +91,7 @@ export function launch(conf) {
     // * close_on_submit: Whether to close modal on clicking submit.
     // * focus_submit_on_open: Whether to focus submit button on open.
     // * help_link: A help link in the heading area.
+    // * id: Custom id to the container element to modify styles.
```

Comment:
This is a great addition, and I'd also do as a preparatory commit before the cutover commit, because it's a commit that would work on top of master.

Reply:
This is intentionally placed after the migration commit since this doesn't apply to Bootstrap modals.

In Micromodal modals, we pass the common id (`dialog_widget_modal`) to the backdrop overlay and the extra id to the actual modal container, whereas in Bootstrap modals, we pass the common id to the container (the backdrop is separately placed in the DOM).
I could add an extra `div` just for this id, but that feels unnecessary.

Reply:
OK. Is the migration commit itself meant to be fully functional, without the commits that follow it?

Reply:
Yup!